.pre-commit-config.yaml

```diff
@@ -7,6 +7,7 @@ repos:
       - id: check-toml
       - id: check-yaml
       - id: check-added-large-files
+        args: ['--maxkb=2000']
```

Comment:
That seems like a really large file. Is it possible to use a text format like WAT [like pecos does](https://github.com/PECOS-packages/PECOS/tree/development/tests/integration/wat)? See https://github.com/WebAssembly/wabt for `wasm2wat`

Additionally, it will help to be able to see the source of the WASM file instead of trusting a binary blob.

Reply:
It's a bit of a bear to use WAT for two reasons: (1) pytket can't load WAT files (2) programmatically converting WAT to wasm files has been proving a bit of a bear without manually invoking wat2wasm which is CLI and would require a lot of awkward shell script usage. I've found that wasmtime can probably make me a byte array, which I would then have to save to a file on the file system as pytket requires a wasm file specifically. All of which is a lot of work to avoid a 1.6MB file I copied from the pytket repository: https://github.com/CQCL/tket/blob/4fae6b93d6f412163e783151a91a1270a4567969/pytket/tests/classical_test.py#L399

Reply:
Fair.

> a lot of work to avoid a 1.6MB file

To be clear it's not just to avoid a big file, but also for readability and version tracking. I always tend to prefer adding a source file instead of a binary when possible.

Reply:
Yeah for sure, I would much prefer wat, but I think that would blow out the scope of this story a bit.